src/main/java/org/maxandersen/starfish/Starfish.java

```diff
@@ -168,6 +169,16 @@ public static void editConfig()throws Exception{
 
 }
 
+//Function to Launch the Editor
+public static void launch_editor(Path directory,String ide,String final_clone_path)throws IOException, InterruptedException{
+//If OS is windows then we add .exe after the command    
+boolean isWindows =System.getProperty("os.name").toLowerCase().indexOf("windows")>=0;
+ide=isWindows?ide+".exe":ide;
+runCommand(directory.getParent(), ide,final_clone_path);//Launching the editor now
+
+}
+
+
 public static void gitClone(Path directory, String originUrl) throws IOException, InterruptedException {
     //Function for git clonning
     runCommand(directory.getParent(), "git", "clone", originUrl, directory.getFileName().toString());
```

Comment:
does this work on windows ? shouldn't it be git.exe too ?

Reply:
This works....It cloned perfectly on my Windows-10 64 bit